static/skin/index.js

```diff
@@ -453,6 +454,24 @@
         }
     }
 
+    function updateNavVisibilityState() {
+        const st = window.scrollY;
+        const scrollPossible = document.querySelector('.kiwixHomeBody').offsetHeight > window.screen.height;
+        const documentWidthIsLessThan590 = document.body.clientWidth < 590;
```

Comment:
There is too much detail in the name of this variable. What if we decide to make the nav bar autohiding on slightly narrower or wider screens? Then either this variable's name will get out of sync with reality or will have to be updated too. It's better to emphasize the meaning/purpose of this variable by naming it something like `enableAutoHiding`.

Reply:
Done.